vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/PageSteps.java

```diff
@@ -248,7 +248,7 @@ public void isPageScrolledToAnElement(Locator locator)
      * <p>
      * @param relativeURL A string value of the relative URL
      */
-    @When("I go to the relative URL '$relativeURL'")
+    @When("I go to relative URL `$relativeURL`")
     public void iGoTo(String relativeURL)
```

Comment:
I think it makes sense to give the method clearer name

Reply:
changed to 'openRelativeUrl'